example/extensions/lib_subgraph/subgraph_lib.cc

```diff
@@ -218,10 +218,27 @@ MXReturnValue mySupportedOps(std::string json,
 
 MXReturnValue myReviewSubgraph(std::string json, int subraph_id, bool* accept,
                                std::unordered_map<std::string, std::string>& options,
-                               std::unordered_map<std::string, std::string>& attrs) {
+                               std::unordered_map<std::string, std::string>& attrs,
+                               std::map<std::string, MXTensor>& args,
+                               std::map<std::string, MXTensor>& aux) {
   for (auto kv : options) {
     std::cout << "option: " << kv.first << " ==> " << kv.second << std::endl;
   }
+  for (auto kv : args) {
```

Comment:
are we missing auxs here ?

Reply:
no, this is just an example implementation of the subgraph library. it can do whatever the user wants it to do.

Reply:
Agreed. I did not notice that this was an example